editor/project_converter_3_to_4.cpp

```diff
@@ -2188,6 +2188,15 @@ bool ProjectConverter3To4::test_conversion(const RegExContainer &reg_container)
 	}
 	valid = valid & (got == expected);
 }
+{
+	String base = "var node = $world/ukraine/lviv.";
+	String expected = "$world/ukraine/lviv.";
```

Comment:
What is this for?

Reply:
This is a test case that validates if `get_object_of_execution` properly handles node path literals.